docs/Reference/Pantheon-API-Methods.md

```diff
@@ -904,6 +904,75 @@ Returns the account balance of the specified address.
     }    
     ```
     
+### eth_getProof
+
+Returns the account and storage values of the specified account, including the merkle proof.
```

Comment:
Did you say this is used by light clients?  If you've got a use-case for when this API would be used, that'd be useful to include

Reply:
Apparently it's used by clients that are unable to even run light clients (IoT devices etc). So I'll add something.